streams/src/test/java/org/apache/kafka/streams/integration/RepartitionOptimizingIntegrationTest.java

```diff
@@ -252,183 +280,183 @@ public void process(final String key, final String value) {
-                                                                + "    Sink: KSTREAM-SINK-0000000038 (topic: joinedOutputTopic)\n"
-                                                                + "      <-- KSTREAM-MERGE-0000000037\n"
+                                                                + "    Source: KSTREAM-SOURCE-0000000013 (topics: [aggregate-groupByKey-repartition])\n"
+                                                                + "      --> aggregate\n"
+                                                                + "    Processor: aggregate (stores: [aggregate-store])\n"
+                                                                + "      --> KTABLE-TOSTREAM-0000000014\n"
+                                                                + "      <-- KSTREAM-SOURCE-0000000013\n"
+                                                                + "    Processor: KTABLE-TOSTREAM-0000000014 (stores: [])\n"
+                                                                + "      --> reduce-to\n"
+                                                                + "      <-- aggregate\n"
+                                                                + "    Sink: reduce-to (topic: outputTopic_1)\n"
+                                                                + "      <-- KTABLE-TOSTREAM-0000000014\n"
                                                                 + "\n"
                                                                 + "  Sub-topology: 2\n"
-                                                                + "    Source: KSTREAM-SOURCE-0000000017 (topics: [KSTREAM-AGGREGATE-STATE-STORE-0000000013-repartition])\n"
-                                                                + "      --> KSTREAM-AGGREGATE-0000000014\n"
-                                                                + "    Processor: KSTREAM-AGGREGATE-0000000014 (stores: [KSTREAM-AGGREGATE-STATE-STORE-0000000013])\n"
-                                                                + "      --> KTABLE-TOSTREAM-0000000018\n"
-                                                                + "      <-- KSTREAM-SOURCE-0000000017\n"
-                                                                + "    Processor: KTABLE-TOSTREAM-0000000018 (stores: [])\n"
-                                                                + "      --> KSTREAM-SINK-0000000019\n"
-                                                                + "      <-- KSTREAM-AGGREGATE-0000000014\n"
-                                                                + "    Sink: KSTREAM-SINK-0000000019 (topic: outputTopic_1)\n"
-                                                                + "      <-- KTABLE-TOSTREAM-0000000018\n"
+                                                                + "    Source: KSTREAM-SOURCE-0000000021 (topics: [reduce-groupByKey-repartition])\n"
```

Comment:
These numbers shouldn't shift at all in this section; they are all increased by 4. Since all state stores and repartition topics are named, I'm not sure this matters, though.  From what I can see, you only added naming to the topology and no operations correct?

Reply:
I added Materialized in-memory stores. I did check to make sure the number of stores before and after was the same, I think the numbers shifted because we are materializing earlier in the topology?